blog/2026-02-25-dev-update-february-2026.md

```
@@ -0,0 +1,213 @@
+
+If data fetching was your bottleneck, this release helps.
+
+**<span style={{ textDecoration: 'underline' }}>25k events per second is now standard</span>**
+
+
+### 📉 Support for DESC Indices
+
+You can now define indices with descending order to improve query performance: 
+
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)